Makefile

```diff
@@ -13,7 +13,7 @@ node_modules: package.json yarn.lock
 	@touch $@
 
 build: ## Builds typescript files and UMD library
-	yarn clean && yarn concurrently "yarn umd" "yarn pkg"
+	yarn clean && yarn build-prep && yarn concurrently "yarn umd" "yarn pkg"
```

Comment:
We need to add a similar prep for the `build-prod` step

Reply:
Thanks for the tip, added!